src/index.html

```diff
@@ -6,8 +6,46 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="page" data-qa="card">
+      <div class="product-card">
+        <div class="product-card-image"></div>
+
+        <div class="product-card-text">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="product-card-code">Product code: 195434</div>
+
+        <div class="product-card-rating">
+          <div class="stars">
```

Comment:
Add the modifier classes here
```suggestion
          <div class="stars stars--4">
```